include/cetl/rtti.hpp

```diff
@@ -35,6 +35,7 @@ using type_id = std::array<std::uint8_t, type_id_size>;
 /// The bytes of the UUID are given as a list of template parameters; there shall be at most 16 of them;
 /// if any are missing, they are assumed to be 0.
 /// For conversion to \ref type_id use \ref cetl::type_id_type_value.
+/// Please don't use empty or all zeros bytes, as it will be treated as the invalid type ID (see \ref type_id_invalid).
```

Comment:
```suggestion
/// Please don't use empty or all zeros bytes.
```

It's a good advise regardless of the issue below.

Reply:
![image](https://github.com/OpenCyphal/CETL/assets/3298404/a29a4703-1e08-4123-9a30-7a128fd86660)